eloquentjs/2_1_looping_a_triangle.js

```diff
@@ -0,0 +1,61 @@
+/**Draws a right triangle utilizing the += operator*/
+function drawTriangle(height) { 
+    if (isNaN(height)) {
+        throw "Triangle height cannot be NaN";
+        return;
+    }
+    if (height <= 0) {
+        throw "Triangle height cannot be zero or less";
+        return;
+    } 
+    let tri = '';
```

Comment:
I'd add a line between your guard clauses and the logic of the function. Using white space helps readers group together different bits of behavior and makes the code more understandable.